packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -210,6 +210,50 @@ export const ContentPanel: FC = () => {
         [currentVizConfig],
     );
 
+    const {
+        tableConfigByChartType,
+        resultsTableRunnerByChartType,
+        handlePivotData,
+    } = useChartResultsTableConfig(resultsRunner, activeConfigs);
```

Comment:
this gets the Line + Bar + Pie charts' table configs. So that we get only the columns for the configured chart. 
For example: 
I choose one x, and on y (sum)

I should only see a table with x and another with y (sum), not all the columns like in the results table.